app/uk/gov/hmrc/nisp/views/includes/helpers/error_overview.scala.html

```diff
@@ -0,0 +1,16 @@
+@(form: Form[_], OptionalErrMsg:Option[String] = None)
+
+@if(form.hasErrors) {
+        <div id="error-summary-display" class="flash error-summary error-summary--show " role="group" aria-labelledby="error-summary-heading" tabindex="-1">
+            <h2 id="error-summary-heading" class="h3-heading">@Messages("nisp.validation.summary.heading")</h2>
+            <ul class="js-error-summary-messages">
+                @form.errors.map { error =>
+                <li role="tooltip">
+                    <a href='#@error.key.replaceAll("""[\.\[\]]""", "_")' id='@error.key.replaceAll("""[\.\[\]]""", "_")-error-summary' data-focuses='@error.key.replaceAll("""[\.\[\]]""", "_")'>
```

Comment:
Why did we add our own error_overview.scala.html and why does it look different to https://github.com/hmrc/play-ui/blob/master/src/main/twirl/uk/gov/hmrc/play/views/helpers/errorSummary.scala.html? Is the regex necessary?